src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.box {
+  display: flex;
+}
+
+.stars {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.normal-star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.active-star {
+  background-image: url(images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
position and repeat are the same for active and non-active stars. So in order not to repeat yourself you should write these styles for .stars selector